task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +360,25 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    let arr = [];
+    let map = {
+        '(': ')',
+        '[': ']',
+        '{': '}',
+        '<': '>'
+    };
+    for (let i = 0; i < str.length; i++) {
+        if (str[i] === '(' || str[i] === '{' || str[i] === '[' || str[i] === '<') {
+            arr.push(str[i]);
+        } else {
+            let last = arr.pop();
+            if (str[i] !== map[last]) {return false};
+        };
+    };
+        if (arr.length !== 0) {
+            return false
+        };
+    return true;
```

Comment:
We are able to return !(arr.length !== 0) with the same result
Please, see https://javascript.info/logical-operators